core/src/main/java/org/mule/runtime/core/internal/policy/PolicyNextActionMessageProcessor.java

```diff
@@ -28,6 +28,7 @@
 import org.mule.runtime.core.api.MuleContext;
 import org.mule.runtime.core.api.context.notification.FlowStackElement;
 import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.api.processor.ContextClassloaderAwareProcessor;
 import org.mule.runtime.core.api.processor.Processor;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.mule.runtime.core.api.processor.Processor'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7602&issues=AWkRT5ZPotIXnmsUSmGc&open=AWkRT5ZPotIXnmsUSmGc)

[ISSUE_KEY]: AWkRT5ZPotIXnmsUSmGc